tests/site_nav.js

```diff
@@ -92,33 +88,44 @@ describe('SiteNav', function() {
     });
   });
 
-  describe('handleChange()', function() {
-    it('should set the list aria hidden label to the opposite of checked',
-        function() {
-      var $toggle = this.$fixture.find('.js-toggle');
-      var $list = this.$fixture.find('#site-menu ul');
-      var testEv = {currentTarget: $toggle};
-
-      this.siteNav.handleChange(testEv);
-      expect($list.attr('aria-hidden')).to.not.equal($toggle.is(':checked'));
+  describe('toggle()', function() {
+    it('should show and hide the menu', function() {
+      var $nav = this.$fixture.find('.js-site-nav');
+      this.siteNav.toggle();
+      expect($nav.attr('class')).to.include('is-open');
```

Comment:
Prefer something like `expect($nav.hasClass('is-open')).to.be.true`. That's a little more robust that checking substring membership, since this logic would also match a hypothetical class like "not-is-open-lol".